src/main/java/com/studyconnect/member/dto/LoginRequest.java

```diff
@@ -1,12 +1,20 @@
 package com.studyconnect.member.dto;
 
 import lombok.Getter;
+import lombok.RequiredArgsConstructor;
 
-/**LoginRequest Domain
+/**
+ * LoginRequest Domain
  * The type LoginRequest.
  */
 @Getter
+@RequiredArgsConstructor
 public class LoginRequest {
 	private String memberEmail;
 	private String password;
+
+	public LoginRequest(String memberEmail, String password) {
+		this.memberEmail = memberEmail;
+		this.password = password;
+	}
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ctor def rcurly' has incorrect indentation level 8, expected level should be 2.